### PR TITLE
[14.0] [mig] edi_storage_oca

### PR DESCRIPTION
**Pull request by fshah-initos**

### CODE REVIEW

File: edi_storage_oca/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -1 +1,2 @@
 * Simone Orsi <simahawk@gmail.com>
+* Foram Shah <foram.shah@initos.com>
```
Comment (simahawk) [inline edi_storage_oca/readme/CONTRIBUTORS.rst:2]: @fshah-initos you can add yourself if here if you provide something more than a bare migration.
TIP: test coverage is very welcomed :wink:

**Review by simahawk — CHANGES_REQUESTED** [review]
edi_oca is missing: mention it in the description and please review https://github.com/OCA/edi/pull/371 to get it merged quickly

**Comment by etobella** [comment]
@fshah-initos Rebase to have edi_oca installed here pleasae :wink:

**Comment by etobella** [comment]
Travis is green finally :smile: However, a rebase would be great in order to have the runbot too

**Comment by fshah-initos** [comment]
All the checks are passed now it is possible to merge the PR.

**Comment by etobella** [comment]
Can you provide better tests please? as commented by @simahawk

**Comment by simahawk** [comment]
> Can you provide better tests please? as commented by @simahawk

Just to be clear: I didn't ask for tests. I just suggested that if you want to figure as contributor you could add tests (or anything else that adds value). BTW this is a well known practice in the OCA. I was just reminding it here :wink:

**Comment by simahawk** [comment]
> Can you provide better tests please? as commented by @simahawk

Just to be clear: I didn't ask for tests. I just suggested that if you want to figure as contributor you could add tests (or anything else that adds value). BTW this is a well known practice in the OCA. I was just reminding it here :wink:

**Comment by simahawk** [comment]
Ops. I pushed the wrong btn, sorry :sweat_smile:

**Comment by fshah-initos** [comment]
Is it possible to merge this PR?

**Review by simahawk — APPROVED** [review]

**Pull request merged**